### DISCUSSION TITLE
give a shadow effect to all articles

### DISCUSSION THREAD

**Discussion by Fre68** [templaza/astroid-framework#329] [Q&A]
Good morning,
(I'm not English -speaking)
Do you know how to give a shadow effect (or box, card) to all articles?
Thanks very much

Frédéric (France)

**Comment by Chacapamac** [comment]
Salut @Fre68, je vais essayer de vous aider.
Ok, Vous voulez ajouter un cadre et une ombre derrière les articles.


Si vous regardez mon site test pour Astroid, en vue ‘Blog” avec la **template Astroid One**, **par défaut**, présente les articles dans un “card-body” div avec cadre et ombre...
—> http://joom4-astro.graf-test.ca/blog

**• Est-ce que ça représente ce que vous voulez ?** 
**• Avez-vous besoin de ce style aussi sur les articles quand ils sont vus séparément ?**
